src/index.html

```diff
@@ -1,14 +1,189 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Human rights</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body>
-  <h1>Human rights</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+     content="width=device-width,
+     user-scalable=no,initial-scale=1.0,
+     maximum-scale=1.0,minimum-scale=1.0"
+     >
+    <meta http-equiv="X-UA-Compatible"
+     content="ie=edge"
+     >
```

Comment:
```suggestion
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
```
if there are fewer than 3 attributes and they take less than 1 line of space, there's no need to put them on new lines